content/docs/13.contributing/02.orchestra.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Orchestra
+---
+
+#### Requirements
+The following dependencies are required to build Kestra locally:
+- Java 17+, Kestra runs on Java 11 but we hit a Java compiler bug fixed in Java 17
+- Node 14+ and npm
+- Python 3, pip and python venv
+- Docker & Docker Compose
+- an IDE (Intellij IDEA, Eclipse or VS Code)
+
+To start contributing:
+- [Fork](https://github.com/kestra-io/kestra/fork) the repository
+- Clone the fork on your workstation:
+
+```shell
+$ git clone git@github.com:{YOUR_USERNAME}/kestra.git
+$ cd kestra
+```
+
+#### Develop backend
+The backend is made with [Micronaut](https://micronaut.io).
+
+Open the cloned repository in your favorite IDE. In most of decent IDEs, Gradle build will be detected and all dependencies will be downloaded.
+You can also build it from a terminal using `./gradlew build`, the Gradle wrapper will download the right Gradle version to use.
+
+- You may need to enable java annotation processors since we are using it a lot.
+- The main class is `io.kestra.cli.App` from module `kestra.cli.main`
+- Pass as program arguments the server you want to develop, for example `server standalone` will the [standalone server](https://kestra.io/docs/administrator-guide/servers/#kestra-standalone-development-environment-servers)
+- ![Intellij Idea Configuration ](https://user-images.githubusercontent.com/2064609/161399626-1b681add-cfa8-4e0e-a843-2631cc59758d.png) Intellij Idea configuration can be found in screenshot below.
```

Comment:
```suggestion

The ![Intellij Idea Configuration ](https://user-images.githubusercontent.com/2064609/161399626-1b681add-cfa8-4e0e-a843-2631cc59758d.png) Intellij Idea configuration can be found in the screenshot below.
```

it would also be better to add images properly in the folder public/docs. Create a folder for your section and add images there